tsconfig.json

```diff
@@ -18,9 +14,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
+    "noImplicitAny": false,
     "jsx": "react-jsx"
   },
-  "include": [
-    "src"
-, "src/styles/AuthButton.tsx"  ]
+  "include": ["src", "src/styles/AuthButton.tsx"]
```

Comment:
hmm I wonder why is `src/styles/AuthButton.tsx` here, maybe it should just include `src`?